src/app/proposals/draft/components/dialogs/SponsorOnchainProposalDialog.tsx

```diff
@@ -0,0 +1,86 @@
+import { VStack } from "@/components/Layout/Stack";
+import { useRouter } from "next/navigation";
+import { useWaitForTransaction } from "wagmi";
+import Image from "next/image";
+import Link from "next/link";
+import { icons } from "@/assets/icons/icons";
+import Tenant from "@/lib/tenant/tenant";
+import { UpdatedButton } from "@/components/Button";
+
+const SponsorOnchainProposalDialog = ({
+  redirectUrl,
+  txHash,
+  closeDialog,
+}: {
+  redirectUrl: string;
+  txHash: `0x${string}`;
+  closeDialog: () => void;
+}) => {
+  const tenant = Tenant.current();
+  const { isLoading } = useWaitForTransaction({
+    hash: txHash,
+  });
+  const router = useRouter();
+  return (
+    <VStack alignItems="items-center">
+      <VStack className="w-full bg-white rounded-xl">
+        <VStack>
+          <VStack className="w-full">
+            <img
+              src={
+                isLoading
+                  ? `/images/action-pending.svg`
+                  : `/images/ens_success.svg`
```

Comment:
The asset images are now in the tenant.ui. You can get them 

const {ui} = Tenant.current()
ui.assets.success
ui.assets.pending